new/LIT6620DerHomKah.xml

```diff
@@ -0,0 +1,77 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6620DerHomKah" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘ፳ወ፬፡ ካህናተ፡ ሰማይ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Dǝrsān za20wa4 kāhǝnāta samāy</title>
+                <title xml:lang="en" xml:id="t2">Homily for the commemoration of the Twenty-Four Elders of Heaven</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Homily"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2022-04-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isDifferentTo" active="LIT6620DerHomKah" passive="LIT6614FourCreat"/>
+                    <relation name="saws:isDifferentTo" active="LIT6620DerHomKah" passive="LIT4958Priests"/>
+                    <relation name="saws:isDifferentTo" active="LIT6620DerHomKah" passive="LIT6469HomilyKahenat"/>
+                    <relation name="ecrm:P129_is_about" active="LIT6620DerHomKah" passive="PRS12519TwentyFour"/>
+                </listRelation>
+            </div><!---->
+
+
+                      <div type="bibliography"/>
```

Comment:
please remove the `div type="bibliography" `around the `div type="edition"`. thank you